### PR TITLE
Aboutme

### PR DESCRIPTION
**Pull request by hannahgolightly**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,17 +48,13 @@ <h1 > hey there! <br/>
...
-					<p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-						labore et dolore magna aliqua. Velit ut tortor pretium viverra suspendisse. Amet nulla facilisi
-						morbi tempus. Ac turpis egestas sed tempus urna et pharetra.
-						Pharetra vel turpis nunc eget lorem dolor sed viverra ipsum. Accumsan in nisl nisi scelerisque.
-						Aliquam etiam erat velit scelerisque in. Penatibus et magnis dis parturient montes nascetur.
-						Lobortis feugiat vivamus at augue eget arcu.<br /> Placerat duis ultricies lacus sed turpis
-						tincidunt id. Eu lobortis elementum nibh tellus molestie nunc non blandit. Aliquet nec
-						ullamcorper sit amet risus. Vitae tortor condimentum lacinia quis vel eros donec. Adipiscing
-						elit duis tristique sollicitudin nibh sit. Ac felis donec et odio. </p>
+					<p> Hi, I&#39m Hannah! After many years working as a Graphic Designer, I decided to make a career change (an idea both exciting and slightly terrifying!). I&#39m currently training to be a full stack developer at Mayden Academy in Bath, a 16 week intensive course to learn the skills required for the industry.
```
Comment (HiredMussel) [inline index.html]: More of a formatting point than anything else, but you might want to split this across multiple lines for the sake of readability

Comment (HiredMussel) [reply index.html]: Also on this line - the validator doesn't like the way you've escaped this character - it seems to want you to put a semicolon immediately before the m

File: style.css [hunk 2]
```diff
@@ -149,11 +149,20 @@ header {
 	
 }
 
-.img_me {
+.img_me_box {
 	margin: 0;
 	width: 33.333%;
+	overflow: hidden;
+
```
Comment (HiredMussel) [inline style.css]: This line is redundant

File: style.css [hunk 3]
```diff
@@ -149,11 +149,20 @@ header {
...
+.img_me {
+	width: 100%;
+	background-image: url("./images/sarah-dorweiler-QeVmJxZOv3k-unsplash.jpg");
 	height: 350px;
-	background-color: #FFE2D1;
+	background-position: center; 
+	background-size: cover; 
+
 }
```
Comment (HiredMussel) [inline style.css:164]: May or may not affect you, but I've had an issue where media queries won't fire on mobile devices. Fix is to add the following element to the head:
<meta name="viewport" content="width=device-width, initial-scale=1">

Comment (HiredMussel) [reply style.css:164]: "<meta name="viewport" content="width=device-width, initial-scale=1">"

Comment (HiredMussel) [reply style.css:164]: "meta name="viewport" content="width=device-width, initial-scale=1"
Sorry, github doesn't like stuff between angle brackets, apparently

**Review by HiredMussel — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (hannahgolightly) [reply style.css:164]: thanks- fixed!

**Pull request merged**